metricsmentor/settings_shared.py

```diff
@@ -72,3 +72,15 @@
 }
 
 COURSEAFFILS_COURSESTRING_MAPPER = CourseStringMapper
+
+DATABASES = {
```

Comment:
You want this to be the case only if the integrationserver is running. Like so: https://github.com/ccnmtl/logiclearner/blob/main/logiclearner/settings_shared.py#L75